examples/surrounding_nitrogen_bonds/random_selection.py

```diff
@@ -0,0 +1,61 @@
+"""Randomly selects N molecules from an input file.
+Usage:
+    python random_selection.py -n [N] --input [FILE] --output [FILE]
+"""
+
+import argparse
+import random
+
+from openeye import oechem
+
+
+def main():
+    """Reads in the file and does the selection."""
+    parser = argparse.ArgumentParser(formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+    parser.add_argument(
+        "-n",
+        type=int,
+        default=20,
+        help="Number of molecules to select.",
+    )
+    parser.add_argument(
+        "--input",
+        type=str,
+        help="Input molecule file.",
+    )
+    parser.add_argument(
+        "--output",
+        type=str,
+        help="Output molecule file.",
+    )
+    parser.add_argument(
+        "--seed",
+        type=int,
+        default=0,
```

Comment:
maybe `default=None` is better? that would default to having no seed at all I believe (though perhaps you don't want that)